cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,132 @@
 /// <reference types='cypress' />
 
+const faker = require('faker');
+
+const firstName = faker.name.firstName();
+const lastName = faker.name.lastName();
+const email = faker.internet.exampleEmail(firstName, lastName);
+const age = faker.random.number({ min: 21, max: 60 });
+const salary = faker.random.number({ min: 2000, max: 7000 });
+const department = faker.commerce.department();
+
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/webtables');
+  });
+
+  it('should be the Pagination', () => {
```

Comment:
```suggestion
  it('should be the Pagination', () => {
```
You need to check buttons next and previous. Add users until 2 page appears, then click next button, assert 2 page opens then click previous button, assert 1 page opens